tests/unit/calendar/helper.js

```diff
@@ -4,6 +4,17 @@ define( [
 ], function( $, helper ) {
 
 return $.extend( helper, {
+	createDate: function( year, month, day ) {
+		if ( arguments.length === 1 ) {
```

Comment:
We shouldn't need to worry about this case for timezone correction, but having all dates go through the helper does make it easier to reason about in the tests.